contributing-guides/git-terminal.md

```diff
@@ -29,3 +31,28 @@ Typically a pull request will include changes in a single file.
 [pr-howto]: ../CONTRIBUTING.md#submitting-a-pull-request
 [commit-msg]: ../CONTRIBUTING.md#commit-message
 [mass-changes]: https://github.com/tldr-pages/tldr/pulls?&q=is:pr+is:merged+label:"mass+changes"
+
+# Updating your fork
+
+Forks aren't updated automatically and in order to avoid merge conflicts it should be done manually and regularly. This is possible via the GitHub web interface by pressing `Fetch upstream` and `Fetch and merge` on the fork:
+
+![Fetch and merge button in GitHub](https://user-images.githubusercontent.com/50295997/119873818-393e2000-bf25-11eb-8a0a-b804c293439b.png).
+
+Alternatively it can also be done using Git in the terminal:
+
+```bash
+git checkout main
+git remote add upstream https://tldr-pages/tldr.git # only run this if you don't already have the upstream remote for https://github.com/tldr-pages/tldr.git
+git fetch upstream main
+git rebase upstream/main
+git push --force # not needed if you only want to update your local repository
+```
+
+# Changing the email of your last commit
+
+If the email that you used for a commit isn't added to your GitHub account, you can either add it [here](https://github.com/settings/emails) or change the email of your last commit with the following commands:
+
+```bash
+git commit --amend --author="Name <new.email@example.com>"
```

Comment:
I think these are the best placeholders here, what do you guys think?
```suggestion
git commit --amend --author="Your Name <your.email@example.com>"
```

Reply:
Why, exactly? I like "Your Name" because it makes it obvious that you can use your display name with spaces in.

Reply:
I like `Your name`, since it shows that you can use spaces, but I prefer `new.email`, so it's obvious that you shouldn't use the original email of the commit.

Reply:
Updated.